### PR TITLE
Test commit

### LINKED ISSUE #2: Duplicated block of code
https://github.com/MoeruDev/rednikeegStudie/blob/d7232cb24e8f0181c4ed286f88d93e43139f03ad/src/main/java/mainpack/MainRun.java#L29-L30
Use do-while loop to avoid this moment

### LINKED ISSUE #3: Centrilize input and output
Those moments should be removed in order to centralize input and output:
https://github.com/MoeruDev/rednikeegStudie/blob/d7232cb24e8f0181c4ed286f88d93e43139f03ad/src/main/java/mainpack/Client.java#L50-L63
https://github.com/MoeruDev/rednikeegStudie/blob/d7232cb24e8f0181c4ed286f88d93e43139f03ad/src/main/java/mainpack/Agreement.java#L51-L64
https://github.com/MoeruDev/rednikeegStudie/blob/d7232cb24e8f0181c4ed286f88d93e43139f03ad/src/main/java/mainpack/Database.java#L13-L20
https://github.com/MoeruDev/rednikeegStudie/blob/d7232cb24e8f0181c4ed286f88d93e43139f03ad/src/main/java/mainpack/Client.java#L63-L71

### LINKED ISSUE #4: Constructors clean up
It's useless to assign nulls inside constructor, because if we haven't assigned value to non-primitive field, then it would be null by default
https://github.com/MoeruDev/rednikeegStudie/blob/d7232cb24e8f0181c4ed286f88d93e43139f03ad/src/main/java/mainpack/Client.java#L37-L44
https://github.com/MoeruDev/rednikeegStudie/blob/d7232cb24e8f0181c4ed286f88d93e43139f03ad/src/main/java/mainpack/Agreement.java#L39-L45

### LINKED ISSUE #6: Refactor switch-case branching
Switch to if expressions
https://github.com/MoeruDev/rednikeegStudie/blob/d7232cb24e8f0181c4ed286f88d93e43139f03ad/src/main/java/mainpack/MainRun.java#L24-L104

### LINKED ISSUE #8: Extract method for string concatenation
Extract those moments as methods to corresponding classes (Client, Agreement) to avoid concatenation of their fields here.
https://github.com/MoeruDev/rednikeegStudie/blob/d7232cb24e8f0181c4ed286f88d93e43139f03ad/src/main/java/mainpack/Database.java#L13-L20
https://github.com/MoeruDev/rednikeegStudie/blob/d7232cb24e8f0181c4ed286f88d93e43139f03ad/src/main/java/mainpack/Client.java#L65-L70
For instance, you can create method getClientInfo() for Client and override method toString() for Agreement.

### PR DESCRIPTION
**Pull request by MoeruDev**

### CODE REVIEW

File: src/main/java/mainpack/InOutHndler.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
+package mainpack;
+
+import java.util.List;
+import java.util.Scanner;
+
+public class InOutHndler {
+
```
Comment (rednikeeg) [inline src/main/java/mainpack/InOutHndler.java]: Typo

File: src/main/java/mainpack/MainRun.java [hunk 2]
```diff
@@ -24,82 +24,69 @@ public void incomeMenu(){
     public static void main(String[] args) {
         MainRun menu = new MainRun();
         Database database = new Database();
+        InOutHndler handler = new InOutHndler();
```
Comment (rednikeeg) [inline src/main/java/mainpack/MainRun.java]: Up to you, but you can have several different handlers at the time, I'd rather renamed this variable to inOutHandler

File: src/main/java/mainpack/MainRun.java [hunk 3]
```diff
@@ -24,82 +24,69 @@ public void incomeMenu(){
...
+        do{
+            menu.mainMenu();
+            choice = scanner.nextInt();
+            if(choice == 1){
+                Client tmp = new Client();
+                handler.addClient(tmp);
+                database.getList().add(tmp);
+                System.out.println("Client successfully added");
+            }
+            else if(choice == 2 || choice == 3 || choice == 4 || choice == 5){
```
Comment (rednikeeg) [inline src/main/java/mainpack/MainRun.java]: Can this expression be simplified with more and less operators?

File: src/main/java/mainpack/InOutHndler.java (hunk 1 above)
Comment (MoeruDev) [reply src/main/java/mainpack/InOutHndler.java]: fixed

File: src/main/java/mainpack/MainRun.java (hunk 3 above)
Comment (MoeruDev) [reply src/main/java/mainpack/MainRun.java]: fixed

**Pull request merged**